### DISCUSSION TITLE
v0.15.1

### DISCUSSION THREAD

**Discussion by ledermann** [solectrus/solectrus#3368] [Updates]
## Minor Changes

- Added `CO2_EMISSION_FACTOR` to allow customization of the CO2 emission calculation (#3290, thanks to @suprnova32)
- Optimized caching, now dependent on time-frame size
- Improved tile loading time on the "Essentials" page
- Added the [Inter](https://rsms.me/inter/) font for consistent appearance across all platforms (significantly better rendering on Windows)
- Updated all dependencies, including an upgrade to Rails 7.2
- Various internal improvements

## Fixes

- Fixed missing values for measurements below 50 Wh (#3358)
- Fixed diagram scaling for consumption and autarky
- Fixed missing translations in the cost tooltip

## New Contributors

- @suprnova32 made their first contribution in https://github.com/solectrus/solectrus/pull/3290

**Full Changelog**: https://github.com/solectrus/solectrus/compare/v0.15.0...v0.15.1

<hr /><em>This discussion was created from the release <a href='https://github.com/solectrus/solectrus/releases/tag/v0.15.1'>v0.15.1</a>.</em>

**Comment by jean13Z** [comment]
> ## Minor Changes
> 
> - Added `CO2_EMISSION_FACTOR` to allow customization of the CO2 emission calculation (#3290, thanks to @suprnova32)
> - Optimized caching, now dependent on time-frame size
> - Improved tile loading time on the "Essentials" page
> - Added the [Inter](https://rsms.me/inter/) font for consistent appearance across all platforms (significantly better rendering on Windows)
> - Updated all dependencies, including an upgrade to Rails 7.2
> - Various internal improvements
> 
> ## Fixes
> 
> - Fixed missing values for measurements below 50 Wh (#3358)
> - Fixed diagram scaling for consumption and autarky
> - Fixed missing translations in the cost tooltip
> 
> ## New Contributors
> 
> - @suprnova32 made their first contribution in https://github.com/solectrus/solectrus/pull/3290
> 
> **Full Changelog**: https://github.com/solectrus/solectrus/compare/v0.15.0...v0.15.1
> 
> <hr /><em>This discussion was created from the release <a href='https://github.com/solectrus/solectrus/releases/tag/v0.15.1'>v0.15.1</a>.</
....
👇
 Press release, run  and code it your way sir. Be sure to accept all responsibility  and effect   in a month. Period usage. Clink. Experimental if nessesar. Thank you..
